src/hooks/use-icons.ts

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from 'react';
+
+// Direct imports
+import AccessIncredibleIPIcon from '/static/images/icons/Access Incredible IP.svg';
```

Comment:
The file names feel inconsistent. Not a good practice to have spaces in file names too.